EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs

```diff
@@ -35,8 +34,8 @@ public void WaitAndClickJonsonDetails(int timeToWait)
         public void WaitAndClickJonsonMenu(int timeToWait)
         {
             _johnsonMenu.WaitAndClick(driver, timeToWait);
-        }
-
+        }                        
```

Comment:
you have empty spaces:
![image](https://user-images.githubusercontent.com/17314644/184872742-8958fb1a-7eab-4812-abb7-f3c7edd1fdeb.png)

please do code cleanup